settings/common.py

```diff
@@ -129,3 +134,17 @@
     ),
 
 }
+
+AUTHENTICATION_BACKENDS = (
+    # Needed to login by username in Django admin, regardless of `allauth`
+    'django.contrib.auth.backends.ModelBackend',
+    # `allauth` specific authentication methods, such as login by e-mail
+    'allauth.account.auth_backends.AuthenticationBackend',
+)
+
+ACCOUNT_AUTHENTICATION_METHOD = "email"
```

Comment:
Let's use `username` as `ACCOUNT_AUTHENTICATION_METHOD` unless you have a specific reason to go for `email`.